phpBB/styles/prosilver/template/attachment.html

```diff
@@ -29,6 +29,26 @@
 		</dl>
 		<!-- ENDIF -->
 
+		{% if _file.S_AUDIO_FILE %}
+			<dl class="file">
+				<dt class="c-file-header">
+					<audio class="c-file-audio-controls" controls="controls" preload="metadata">
+						<source class="c-file-audio-source" src="{{ _file.U_DOWNLOAD_LINK }}" type="{{ _file.MIMETYPE }}">
```

Comment:
The `class="c-file-audio-source"` is not defined in the CSS.

Reply:
Which is fine, there is currently no need to style it but with this being declared one could style it if one would want to.